css/index.css

```diff
@@ -143,4 +176,133 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/*My code starts vhere*/
+
+/*header*/
+header {
+    width: 800px;
+    margin: 0 auto;
+}
+/* nav section*/
+nav {
+    disooter nav
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+
+}
+
+footer nav a {
+justify-content: flex-end;
+margin-top: 20px;
+}
+    jusooter nav{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+
+}
+
+footer nav a {
+justify-content: flex-end;
+margin-top: 20px;
+}
+    fleooter nav{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+
+}
+
+footer nav a {
+justify-content: flex-end;
+margin-top: 20px;
+}
+   
+    
+nav{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+}
+
+nav a {
+justify-content: flex-end;
+margin-top: 20px;
+text-decoration:none;
+color: black;
+justify-content: flex-end;
+margin-top: 20px;
+}
+
+/*#logo{
+    justify-content: flex-start;
+    
+}*/
+
+#headerimage{
+    margin-top: 50px;
+    width: 1200px;
+    height: auto;
+    margin-bottom: 30px;
+    }
+/*"About" page starts here*/
+
+#about-container{
+    display:flex;
+    flex-wrap: wrap;
+    width: 800px;
+    margin: 0 auto;
+    margin-bottom: 30px;
+    
+    padding: 20px;
+      
+}
+#about-text{
+    border-bottom: 1px dashed black;
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+/*#about-text p {
+    margin-bottom: 20px;
+}*/
+
+
+.body-images{
+    display: inline-flex;
```

Comment:
So with inline-flex, it turns your flex container into an inline element as opposed to a block element with display : flex.  that is the only difference apparently.  There is no changes to how the flex items behave.

Reply:
https://stackoverflow.com/questions/27418104/whats-the-difference-between-displayinline-flex-and-displayflex